compute/oslogin/service_account_ssh.py

```diff
@@ -146,16 +146,23 @@ def main(
         '--cmd', default='uname -a',
         help='The command to run on the remote instance.')
     parser.add_argument(
-        '--project', default='my-project-id',
+        '--project',
         help='Your Google Cloud project ID.')
     parser.add_argument(
-        '--zone', default='us-central1-f',
+        '--zone',
         help='The zone where the target instance is locted.')
     parser.add_argument(
-        '--instance', default='my-instance-name',
+        '--instance',
         help='The target instance for the ssh command.')
+    parser.add_argument(
+        '--account',
+        help='The service account email.')
+    parser.add_argument(
+        '--hostname',
+        help='The external IP address or hostname for the target instance.')
     args = parser.parse_args()
 
-    main(args.cmd, args.project, args.instance, args.zone)
+    main(args.cmd, args.project, args.instance, args.zone,
```

Comment:
Should this be instance=args.instance, zone=args.zone here?

Reply:
Good catch this could fail in some situations. Thanks!